Vocable/SwiftUI/EditCategoryDetail.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  ExampleVocableView.swift
```

Comment:
Comment seems to still have the old file name

```suggestion
//  EditCategoryDetail.swift
```